bundles/AdminBundle/Controller/Admin/DataObject/ClassificationstoreController.php

```diff
@@ -817,6 +826,7 @@ public function relationsActionGet(Request $request)
         if ($limit > 0) {
```

Comment:
why not move this up to the $request->getLimit path (line 815) ?

Reply:
I dont understand. The limit will only be set if it is greater than 0.

`$list` does not exist in line 815 because it is initialized in line 824.

Reply:
as far as I see limit is set to 15 by default in line 788 and only overwritten if passes as a request parameter. 
but let's leave it as it is for, should be good enough ;-)
any objections ?

Reply:
Nope. Refactoring potential exists in some places 🤔 
But you can't change too much at once with so many contributors. Because there are usually merge conflicts.

Otherwise I should already send a few PSR12 PRs ^^